server/app.js

```diff
@@ -25,6 +25,9 @@ require('./config/express')(app, authenticate);
 // Link routes
 require('./config/routes').initialise(app);
 
+// Load and configure plugins
+require('../plugins/pluginManager')(app, config);
```

Comment:
The plugin manager exposed function is called with a full configuration object, I should send only the plugin property as long as you are using only plugin property at the plugin manager .